etc/optimus.yaml

```diff
@@ -49,6 +49,12 @@ workers:
     # Optional.
     # Default value is false.
     # dry_run: true
+    # Identity level specifies which kind of customers this bot allows to work
+    # with.
+    # Required.
+    # Possible values are: unknown, anonymous, registered, identified and
```

Comment:
I don't think we should mention `unknown` in comments. Moreover - check for non-zero value is a good idea

Reply:
Ah, it's already required, so should not be zero. Then remove `unknown` from comment please